### PR TITLE
Remove extra setTimeout in Blockchain#connect

### PR DESCRIPTION
**Pull request by fanatid**
Hello `const`/`for..of`/`spread`! 😆

### CODE REVIEW

**Comment by dcousens** [comment]
LGTM,  but what was the rationale?
I was going to say it changes the memory requirements,  but I suppose they are the same.

File: blockchain.js [hunk 1]
```diff
@@ -25,28 +26,33 @@ Blockchain.prototype.connect = function (blockId, height, callback) {
...
     debug(`Putting ${blockId} @ ${height} - ${transactions.length} transactions`)
     atomic.put(types.tip, {}, { blockId, height })
     atomic.write((err) => {
       if (err) return callback(err)
 
-      this.connect2ndOrder(blockId, block, (err) => callback(err, block.nextblockhash))
+      this.connect2ndOrder(blockId, block, (err) => {
+        callback(err, block.nextblockhash)
+        setTimeout(() => {
+          for (const args of events) this.emitter.emit(...args)
```
Comment (dcousens) [inline blockchain.js:53]: `.forEach`?  if we want to `for of`... maybe do it everywhere in another PR?
Also,  what was the minimum node version for this?

Comment (fanatid) [reply blockchain.js:53]: Minimum 6.0.0 as I know

**Comment by fanatid** [comment]
I think memory requirements will be same, because instead array you store arguments in v8 queue (or somewhere in node).
IDC about PR, feel free drop/change/etc. It was first thing which I wanted to change when looked on code.

**Review by dcousens — APPROVED** [review]
